openfe_gromacs/protocols/gromacs_md/md_methods.py

```diff
@@ -0,0 +1,649 @@
+
+        return gro
+
+    def get_top_filename(self) -> list[pathlib.Path]:
+        """
+        Get a list of paths to the .gro file
+
+        Returns
+        -------
+        traj : list[pathlib.Path]
+          list of paths (pathlib.Path) to the simulation trajectory
+        """
+        top = [pus[0].outputs["system_top"] for pus in self.data.values()]
+
+        return top
+
+    def get_mdp_filenames(self) -> list[list[pathlib.Path]]:
+        """
+        Get a list of paths to the .mdp files
+
+        Returns
+        -------
+        mdps : list[list[pathlib.Path]]
+          list of paths (pathlib.Path) to the mdp files for energy minimization,
+          NVT and NPT MD runs
+        """
+        mdps = [pus[0].outputs["mdp_files"] for pus in self.data.values()]
+
+        return mdps
+
+
+class GromacsMDProtocol(gufe.Protocol):
+    """
+    Protocol for running Molecular Dynamics simulations using Gromacs.
+
+    See Also
+    --------
+    :mod:`openfe.gromacs.protocols`
+    :class:`openfe.gromacs.protocols.gromacs_md.GromacsMDProtocolSettings`
+    :class:`openfe.gromacs.protocols.gromacs_md.GromacsMDProtocolUnit`
+    :class:`openfe.gromacs.protocols.gromacs_md.GromacsMDProtocolResult`
+    """
+
+    result_cls = GromacsMDProtocolResult
+    _settings: GromacsMDProtocolSettings
+
+    @classmethod
+    def _default_settings(cls):
+        """A dictionary of initial settings for this creating this Protocol
+
+        These settings are intended as a suitable starting point for creating
+        an instance of this protocol.  It is recommended, however that care is
+        taken to inspect and customize these before performing a Protocol.
+
+        Returns
+        -------
+        Settings
+          a set of default settings
+        """
+        return GromacsMDProtocolSettings(
+            forcefield_settings=settings.OpenMMSystemGeneratorFFSettings(
+                nonbonded_method="PME",
+                constraints=None,
+            ),
+            thermo_settings=settings.ThermoSettings(
+                temperature=298.15 * unit.kelvin,
+                pressure=1 * unit.bar,
+            ),
+            partial_charge_settings=OpenFFPartialChargeSettings(),
+            solvation_settings=OpenMMSolvationSettings(),
+            engine_settings=OpenMMEngineSettings(),
+            integrator_settings=IntegratorSettings(),
+            simulation_settings_em=EMSimulationSettings(
+                integrator="steep",
+                nsteps=5000,
+                tcoupl="no",
+                pcoupl="no",
+                gen_vel="no",
+            ),
+            simulation_settings_nvt=NVTSimulationSettings(
+                nsteps=50000,  # 100ps
+                pcoupl="no",
+                gen_vel="yes",
+            ),
+            simulation_settings_npt=NPTSimulationSettings(
+                nsteps=500000,  # 1ns
+                pcoupl="Parrinello-Rahman",
```

Comment:
PR can be very flaky if you start from a system that is far from the target pressure - I've at least encountered this a few times in the past even in soluble proteins.

Personally I'd change the default to C-rescale, it's a lot more stable and equally accurate.

Reply:
Changed this!